homeassistant/components/netatmo/__init__.py

```diff
@@ -79,6 +95,28 @@
             hass.config_entries.async_forward_entry_setup(entry, component)
         )
 
+    webhook_id = secrets.token_hex()
+    data = hass.data[DOMAIN][entry.entry_id]
+
+    if hass.components.cloud.async_active_subscription():
+        data[CONF_CLOUDHOOK_URL] = await hass.components.cloud.async_create_cloudhook(
```

Comment:
You're now creating a cloudhook at each startup. You should create a webhook ID during the config flow and make it a cloudhook at that point in time. Store that url in entry.data. You could do a one time migration for existing config entries now.